e2e/pages/mentors.page.ts

```diff
@@ -0,0 +1,24 @@
+import { Locator, Page } from "@playwright/test"
+
+export class MentorsPage {
+  readonly page: Page
+  readonly mentorListName: Locator
+  readonly searchBar: Locator
+  readonly loadingProgress: Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.mentorListName = page.locator(".font-semibold.tracking-tight.text-2xl")
+    this.searchBar = page.locator("#search")
+    this.loadingProgress = page.locator(".dark.nprogress-busy")
```

Comment:
Please use the `getBy` query as much as possible, following [this priority](https://testing-library.com/docs/queries/about/#priority).
```suggestion
    this.loadingProgress = page.getByRole("progressbar")
```

Reply:
updated https://github.com/bearmentor/bearmentor/pull/8/commits/5d078d7fb66a9ce09a715fbb88b0c046ed7a9f44